### ISSUE TITLE
❓ Question: how to edit the homepage and port

### ISSUE THREAD

**Issue by coaz** [stonith404/pingvin-share#102]
### 🙋‍♂️ Question

I want to edit the home page and port of the app. Can someone help me?

This software has to be compiled via an IDE or just modifying the code?

**Comment by stonith404** [comment]
Did you install Pingvin Share with Docker or stand-alone?

**Comment by stonith404** [comment]
The edit the home page change the following file: `frontend/src/pages/index.tsx`. To apply the changes you have to rebuild the frontend with `npm run build` and restart it. See the setup instructions.

To change the port of the frontend, run 
```bash
pm2 start --name="pingvin-share-frontend" npm -- run start -- -p <port>
# instead of pm2 start --name="pingvin-share-frontend" npm -- run start
```

**Comment by stonith404** [comment]
Did I answer your question, if yes could you please close the issue?

**Comment by coaz** [comment]
Yes, thank you. I achieve it but I can’t use port 80. Is something in my server?

**Comment by stonith404** [comment]
I don't know if something is running on your server on port 80 as I don't have access to it. You can try to identify which service is using port 80.[ Here is a link how you can do that](https://unix.stackexchange.com/questions/106561/finding-the-pid-of-the-process-using-a-specific-port).

**Comment by coaz** [comment]
![imagen](https://user-images.githubusercontent.com/27463067/221425817-de9e9cef-485a-44ca-ab3e-f42027f53528.png)
It display that the port is free but I can't run pingvin in this port.

**Comment by stonith404** [comment]
What are the logs of the frontend? 
```
pm2 logs pingvin-share-frontend
```
